gitee/payload.go

```diff
@@ -0,0 +1,289 @@
+package gitee
+
+import (
+	"time"
+)
+
+type CommentEventPayload struct {
+	Action        *string          `json:"action,omitempty"`
+	Comment       *NoteHook        `json:"comment,omitempty"`
+	Repository    *ProjectHook     `json:"repository,omitempty"`
+	Project       *ProjectHook     `json:"project,omitempty"`
+	Author        *UserHook        `json:"author,omitempty"`
+	Sender        *UserHook        `json:"sender,omitempty"`
+	URL           *string          `json:"url,omitempty"`
+	Note          *string          `json:"note,omitempty"`
+	NoteableType  *string          `json:"noteable_type,omitempty"`
+	NoteableID    int64            `json:"noteable_id,omitempty"`
+	Title         *string          `json:"title,omitempty"`
+	PerIID        *string          `json:"per_iid,omitempty"`
+	ShortCommitID *string          `json:"short_commit_id,omitempty"`
+	Enterprise    *EnterpriseHook  `json:"enterprise,omitempty"`
+	PullRequest   *PullRequestHook `json:"pull_request,omitempty"`
+	Issue         *IssueHook       `json:"issue,omitempty"`
+	HookName      *string          `json:"hook_name,omitempty"`
+	Password      *string          `json:"password,omitempty"`
+}
+
+type PushEventPayload struct {
+	Ref                *string         `json:"ref,omitempty"`
+	Before             *string         `json:"before,omitempty"`
+	After              *string         `json:"after,omitempty"`
+	TotalCommitsCount  int64           `json:"total_commits_count,omitempty"`
+	CommitsMoreThanTen *bool           `json:"commits_more_than_ten,omitempty"`
```

Comment:
The same thing about bool! Do we actually need to distinguish false and nil everywhere?